components/cdc/src/endpoint.rs

```diff
@@ -800,6 +818,7 @@ impl<T: 'static + RaftStoreRouter> Runnable<Task> for Endpoint<T> {
             Task::MultiBatch { multi } => self.on_multi_batch(multi),
             Task::OpenConn { conn } => self.on_open_conn(conn),
             Task::RegisterMinTsEvent => self.register_min_ts_event(),
+            Task::ReadCallback { resp, cb } => cb(resp),
```

Comment:
Could you name it `InitializeDownstream` or something similar and call `uninitialized_to_normal` here?